telegram/src/main/kotlin/me/ivmg/telegram/entities/BotCommand.kt

```diff
@@ -0,0 +1,9 @@
+package me.ivmg.telegram.entities
+
+import com.google.gson.annotations.SerializedName as Name
+import me.ivmg.telegram.entities.stickers.ChatPermissions
```

Comment:
Really minor: unused imports here.

If you run the `./gradlew ktlintFormat` command, `ktlint` will take care of this things for you.